qa/acceptance-tests/src/test/java/io/camunda/it/util/TestHelper.java

```diff
@@ -608,6 +609,36 @@ public static void waitUntilJobWorkerHasFailedJob(
             });
   }
 
+  public static void waitUntilJobHasBeenActivated(
+      final CamundaClient camundaClient, final Long jobKey, final int expectedJobs) {
+    Awaitility.await("should wait until job has been activated")
+        .atMost(TIMEOUT_DATA_AVAILABILITY)
+        .ignoreExceptions() // Ignore exceptions and continue retrying
+        .untilAsserted(
+            () -> {
+              final var result =
+                  camundaClient.newJobSearchRequest().filter(f -> f.jobKey(jobKey)).send().join();
```

Comment:
❓ Does this really wait until the job has been activated? It currently only waits until it can be found in general, right?